easybuild/easyconfigs/g/GPAW-setups/GPAW-setups-0.9.9672.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'Tarball'
+
+name = 'GPAW-setups'
+version = '0.9.9672'
+
+homepage = 'https://wiki.fysik.dtu.dk/gpaw/'
+description = """PAW setup for the GPAW Density Functional Theory package.  
+Users can install setups manually using 'gpaw install-data' or use setups from this package.  
+The versions of GPAW and GPAW-setups can be intermixed."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+source_urls = ['https://wiki.fysik.dtu.dk/gpaw-files/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['f0907195df141365cbcc76159b78870e33e45688ac6886640990e153430712ce']
+
+modextrapaths = {'GPAW_SETUP_PATH': ''}
+
+moduleclass = 'chem'
```

Comment:
This should be the last line of the easyconfig file (same goes for the other `GPAW-setups` easyconfigs